src/components/MyList/MyList.tsx

```diff
@@ -0,0 +1,34 @@
+import { memo } from 'react';
+import { User } from '../../models/user';
+import { MyListItem } from '../MyListItem/MyListItem';
+import './MyList.css';
+
+interface MyListProps {
+  users: User[];
+}
+
+function MyList(props: MyListProps) {
```

Comment:
Always specify the return type of the components

```suggestion
function MyList(props: MyListProps): JSX.Element {
```

Reply:
done